content/working_with/console/widgets/sitesMap.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: bt_wiki
+title: Sites Map
+category: Cloudify Console
+draft: false
+---
+Displays the world map with defined sites marked. Only sites with defined location are displayed on the map.
+
+![sitesMap]( /images/ui/widgets/sitesMap.png )
+
+Color of the site marker on the map indicates site health. It is derived from status of deployments attached to the site.
+ 
+After clicking on site marker on the map, user can see popup with site details.
```

Comment:
The color of the site marker on the map indicates the site's health. It is derived from the status of the deployments attached to the site.

After clicking the site marker on the map, the user is presented with a popup with the status breakdown of the deployments assigned with the site.

Reply:
Done.